pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -770,7 +770,7 @@ func (cpdb *MySQLCheckpointsDB) Get(ctx context.Context, tableName string) (*Tab
 		tableRow := tx.QueryRowContext(c, tableQuery, tableName)
 
 		var status uint8
-		if err := tableRow.Scan(&status, &cp.AllocBase, &cp.TableID); err != nil {
+		if err := tableRow.Scan(&status, &cp.AllocBase, &cp.TableID); err != nil && err != sql.ErrNoRows {
```

Comment:
Why need to skip the `ErrNoRows` error. I think it's better to check this error explicitly if caller need to skip it.

Reply:
https://github.com/pingcap/br/blob/master/pkg/lightning/checkpoints/checkpoints_file_test.go#L217

for the file driver, it will return a Missing `CheckpointStatusMissing` status checkpoint, we should keep the same behavior.

Also, the users should not care about the different implementations of the checkpoint DB.

Reply:
Since only the check empty tables logic depends on the get operation before checkpoint db is inited, I think you can make both MySQL and file checkpoint return nil or something like NotFound error to represent that this checkpoint data do not exist. In other normal logic, I don't think it's a good idea to just ignore it.

Reply:
> Since only the check empty tables logic depends on the get operation before checkpoint db is inited, I think you can make both MySQL and file checkpoint return nil or something like NotFound error to represent that this checkpoint data do not exist. In other normal logic, I don't think it's a good idea to just ignore it.

then what's the usage of `CheckpointStatusMissing` status?